homeassistant/components/zwave_js/light.py

```diff
@@ -290,8 +291,10 @@ def _calculate_color_values(self) -> None:
                 and green_val.value is not None
                 and blue_val.value is not None
             ):
-                self._hs = color_util.color_RGB_to_hs(
-                    red_val.value, green_val.value, blue_val.value
+                self._hs_color = list(
```

Comment:
This is a typing correction.

Reply:
Can we make `self._hs_color` a tuple ?

Reply:
Yeah, that's better. The service schema validates the hs color to a tuple. The background was the hs color docstring in the light entity which hints at a list.